.github/workflows/test.yml

```diff
@@ -27,6 +27,9 @@ jobs:
       run: pip install -e '.[linting,testing]' --extra-index-url https://download.pytorch.org/whl/cpu
     - name: Repo line count
       run: python3 sz.py
+    - name: EditorConfig Checker
+      uses: editorconfig-checker/action-editorconfig-checker@main
+      run: editorconfig-checker
```

Comment:
Nope, the CI didn't run cuz the workflow file is broken, can't put the uses and the run into the same step.